sentences/fr/media_player_HassMediaUnpause.yaml

```diff
@@ -0,0 +1,9 @@
+language: fr
+intents:
+  HassMediaUnpause:
+    data:
+      - sentences:
+          - "lecture sur [<le>]{name}"
+          - "Reprend[s] sur [<le>]{name}"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms